spk/demowebservice/Makefile

```diff
@@ -43,7 +46,7 @@ CONF_DIR = src/conf_php$(PHP_VERSION)/
 # Disable the creation of app/config file:
 NO_SERVICE_SHORTCUT = y
 # create firewall rules, since we add a port based "Web Portal"
-SERVICE_PORT = 8889
+SERVICE_PORT = 18889
 endif
```

Comment:
Is there a need to update the wiki?  Friendly reminder just in case.

Reply:
> Is there a need to update the wiki? Friendly reminder just in case.

I added a comment that it conflicts with a port used by Synology SRM and will update it when this is merged.
I updated the wiki with ports additionally used by Synology SRM
https://github.com/SynoCommunity/spksrc/wiki/Synology-Used-Ports